pkg/asset/installconfig/alibabacloud/alibabacloud.go

```diff
@@ -18,11 +18,42 @@ const (
 	defaultAcceptLanguage = "en-US"
 )
 
-var unsupportedRegions = sets.NewString(
-	// Nanjing is a local cloud
-	"cn-nanjing",
+var supportedRegions = sets.NewString(
+	"cn-qingdao",
+	"cn-beijing",
+	"cn-zhangjiakou",
+	"cn-huhehaote",
+	"cn-wulanchabu",
+	"cn-hangzhou",
+	"cn-shanghai",
+	"cn-shenzhen",
+	"cn-heyuan",
+	"cn-guangzhou",
+	"cn-chengdu",
+	"cn-hongkong",
+	"ap-northeast-1",
+	"ap-southeast-1",
+	"ap-southeast-2",
+	"ap-southeast-3",
+	"ap-southeast-6",
+	"ap-southeast-5",
+	"ap-south-1",
+	"us-east-1",
+	"us-west-1",
+	"eu-west-1",
+	"eu-central-1",
+
+	// Nanjing, Fuzhou is a local cloud
```

Comment:
Do you mean "Nanjing and Fuzhou are local regions"?

Reply:
aHa~, yes, and I have updated this comment